lang/ui.ca.json

```diff
@@ -0,0 +1,1161 @@
+  },
+  "simulator-radio-off": {
+    "defaultMessage": "La ràdio està tancada",
+    "description": "Text shown when there are no radio messages because the radio is off in the simulator user interface"
+  },
+  "simulator-radio-send": {
+    "defaultMessage": "Envia missatge",
+    "description": "Aria label for the simulator radio send button"
+  },
+  "simulator-radio-user": {
+    "defaultMessage": "Has enviat:",
+    "description": "Visually hidden text for a radio message sent from the user to the simulated micro:bit. Text of the message follows."
+  },
+  "simulator-reference-link": {
+    "defaultMessage": "Enllaç a la secció de Referència",
+    "description": "Aria label for simulator Reference link button"
+  },
+  "simulator-reset": {
+    "defaultMessage": "Reiniciar",
+    "description": "Aria label for the reset simulator button"
+  },
+  "simulator-serial-terminal": {
+    "defaultMessage": "Terminal sèrie del simulador",
+    "description": "Aria label for the simulator serial terminal"
+  },
+  "simulator-sound-level": {
+    "defaultMessage": "Nivell de so",
+    "description": "Sound level simulator panel title"
+  },
+  "simulator-start-simulator": {
+    "defaultMessage": "Inicia el simulador",
+    "description": "Aria label for the large play button on the simulator board"
+  },
+  "simulator-stop": {
+    "defaultMessage": "Atura el simulador",
+    "description": "Aria label for the stop simulator button"
+  },
+  "simulator-temperature": {
+    "defaultMessage": "Temperatura",
+    "description": "Temperature simulator panel title"
+  },
+  "simulator-title": {
+    "defaultMessage": "Simulador",
+    "description": "Simulator title"
+  },
+  "simulator-touch-logo": {
+    "defaultMessage": "Logotip tàctil",
+    "description": "Name for the touch logo pin on the V2 micro:bit board. Used in the simulator."
+  },
+  "simulator-unmute": {
+    "defaultMessage": "Deixar de silenciar",
+    "description": "Aria label for the unmute simulator button"
+  },
+  "software-versions": {
+    "defaultMessage": "Versions de programari",
+    "description": "Heading for a table of software versions in the about dialog"
+  },
+  "start-coding-action": {
+    "defaultMessage": "Comença a codificar",
+    "description": "Start coding button text"
+  },
+  "support": {
+    "defaultMessage": "Suport",
+    "description": "Support menu option text"
+  },
+  "terms-of-use": {
+    "defaultMessage": "Condicions d'ús",
+    "description": "Terms of use menu option text"
+  },
+  "third-party-module-explanation": {
+    "defaultMessage": "Aquest fitxer és un mòdul de tercers i no està pensat per ser editat.",
+    "description": "Explanation shown instead of the code editor for third-party modules."
+  },
+  "third-party-module-how-to": {
+    "defaultMessage": "Permet editar mòduls de tercers a <link>Settings</link> (configuració) per modificar aquest mòdul.",
```

Comment:
Settings needs translating to match the key string with id "settings".
Currently "Configuració"